src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -30,6 +30,7 @@
     <XunitCommandLine>xunit.console.netcore.exe $(TargetFileName)</XunitCommandLine>
     <XunitOptions>$(XunitOptions) -xml $(XunitResultsFileName)</XunitOptions>
     <XunitOptions>$(XunitOptions) -notrait category=failing</XunitOptions>
+    <XunitOptions>$(XunitOptions) -notrait category=nonwindowstests</XunitOptions>
```

Comment:
This excludes tests meant only for linux and mac to not be run on windows. Since for now, the build script for linux and mac is not up, i dunno what are the variables that will be used to condition the platform. Once that support is up, i will condition this, but for now, this script is run only on windows.